models/engine/db_storage.py

```diff
@@ -71,6 +71,22 @@ def reload(self):
         Session = scoped_session(sess_factory)
         self.__session = Session
 
+    def get(self, cls, id):
+        """Retrieve object by class and id
+        """
+        if cls in classes.values():
+            return self.__session.query(cls).filter(cls.id == id).first()
+        else:
+            return None
+
+    def count(self, cls=None):
+        """Count number of objects in storage
+        """
+        if cls:
+            return len(self.all(cls))
```

Comment:
Same here. If you wanted this to be shorter, this is technically the only line needed.